src/components/NavigationGroup/NavigationGroup.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../index.scss';
+
+.navigationGroupContainer{
```

Comment:
Can you change the name of HTML classes so they are named according to this convention http://wiki.c2.com/?KebabCase